examples/solidstart-minimal-starter/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "solidstart-minimal-starter",
+  "scripts": {
+    "dev": "solid-start dev",
+    "build": "solid-start build",
+    "start": "solid-start start"
+  },
+  "type": "module",
+  "devDependencies": {
+    "solid-start-node": "^0.2.15",
+    "typescript": "^4.8.3",
+    "vite": "^3.1.3"
+  },
+  "dependencies": {
+    "@adeora/solid-query": "^0.12.0",
```

Comment:
Why not tanstack?

Reply:
it is a patched package that supports ssr published by the creator of tanstack solid query (aryan),
he published it so i could add ssr support to trpc,
that is because ssr will be supported under the tanstack org only in version 5 (which may take a while)

Reply:
https://github.com/orjdev/solid-trpc

https://github.com/orjdev/solid-trpc/tree/ssr

therefore it is labeled under the `@ssr` tag for now